meta/main.yml

```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: "Disaster recovery for oVirt engine"
   company: Red Hat, Inc.
   license: Apache License 2.0
-  min_ansible_version: 2.7.2
+  min_ansible_version: 2.9
```

Comment:
We should also require 2.9 version on the spec file - 
ovirt-ansible-disaster-recovery.spec.in

Reply:
Done, thanks!